src/test/java/org/broadinstitute/hellbender/utils/recalibration/RecalDatumUnitTest.java

```diff
@@ -97,12 +97,12 @@ public void testRecalDatumCopyAndCombine(RecalDatumTestProvider cfg) {
     @Test(dataProvider = "RecalDatumTestProvider")
     public void testRecalDatumModification(RecalDatumTestProvider cfg) {
         RecalDatum datum = cfg.makeRecalDatum();
-        datum.setEmpiricalQuality(10.1);
-        Assert.assertEquals(datum.getEmpiricalQuality(), 10.1);
+        datum.setEmpiricalQuality(10); // tsato: 10.1 -> 10. Downstream data might be affected.
```

Comment:
Does changing the quality from double -> int potentially change the results of the recalibration table by rounding earlier?  I'm not sure what the advantage of that is?